src/main/resources/application.yml

```diff
@@ -20,6 +21,7 @@ spring:
     username: admin
     password: password
     driverClassName: org.h2.Driver
+    initialization-mode: always
```

Comment:
This allows the data script below to run properly. I saw your `@PostConstruct` code for the users table. I'm open to using that approach for data population. Feel free to let me know your opinion and I can update.